engine/class_modules/paladin/sc_paladin.hpp

```diff
@@ -917,6 +940,11 @@ struct paladin_action_t : public Base
       am *= 1.0 + p() -> buffs.avenging_wrath -> get_damage_mod();
     }
 
+    if ( affected_by.avenging_crusader && p()->buffs.avenging_crusader->up() )
+    {
+      am *= 1.0 + p() -> buffs.avenging_crusader -> get_damage_mod();
+    }
+
```

Comment:
With the changes I proposed earlier, this would turn into 
```suggestion
    if ( affected_by.avenging_crusader )
    {
      am *= 1.0 + p() -> buffs.avenging_crusader -> check_value();
    }

```

Reply:
i dislike removing the up part but i'll test it

Reply:
Tested and works great. Love you